### DISCUSSION TITLE
Want to see the max_sample_age of servers

### DISCUSSION THREAD

**Discussion by bat-manav** [zubkov-andrei/pg_profile#40] [Q&A]
Hi,

I used set_server_max_sample_age to set the retention period of a server. However I am not getting any option to view the max_sample_age settings. 
show_servers function not giving this inform.

Is there any function available to view this information. This will be useful. I am logging to a database for first time and want to check the existing settings.

Regards
Rajkishore
![Screenshot 2022-06-01 at 6 25 37 PM](https://user-images.githubusercontent.com/68768446/171409342-25b334e8-e74e-4106-ae73-eba7dc1b5dc4.png)

**Comment by zubkov-andrei** [comment]
Finally implemented in 4.2
